build/package/nfpm.yaml

```diff
@@ -42,6 +42,7 @@ contents:
   dst: /lib/systemd/system/kong.service
 - src: build/package/kong.logrotate
   dst: /etc/kong/kong.logrotate
+  type: config|noreplace
```

Comment:
let's also add `config` (without noreplace) to /etc/kong/kong.conf.default
```yaml
- src: nfpm-prefix/etc/kong
  dst: /etc/kong
```
to be replaced by 1) /etc/kong type dir, 2) /etc/kong/kong.conf.default type config

Reply:
done